src/appleseedmaya/exporters/shadingnodeexporter.cpp

```diff
@@ -469,18 +469,9 @@ void ShadingNodeExporter::exportValue(
     }
     else if (paramInfo.paramType == "float")
     {
-        if (paramInfo.mayaAttributeType == "angle")
```

Comment:
There are 2 nodes I think. place2d node and maya anisotropic material.
We used the metadata for them before.